20_kata.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/55c6126177c9441a570000cc
+
+import re
+
+
+def order_weight(strng):
+
+    cleaned_string = re.findall(r"\d+", strng)
+    cleaned_string = sorted(cleaned_string)
+
+    def sum_of_digit(digits):
+        separated = list(digits)
```

Comment:
This variable could have better name, but is also not needed. You can iterate over string characters just like you iterate over elements of the list. Please delete it.